### PR TITLE
ISPN-13032 Simple tutorial with encoding change example

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-13032

### CODE REVIEW

File: infinispan-remote/encoding/src/main/java/org/infinispan/tutorial/simple/encoding/InfinispanEncodingCaches.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      URI textCacheURI = InfinispanEncodingCaches.class.getClassLoader().getResource("textCache.xml").toURI();
+
+      builder.remoteCache("textCache")
+            .configurationURI(textCacheURI).marshaller(new UTF8StringMarshaller());
+
+      RemoteCacheManager cacheManager = new RemoteCacheManager(builder.build());
+      RemoteCache<String, Object> textCache = cacheManager.getCache("textCache");
+
+      System.out.println("== Cache with text encoding and text marshaller.");
+      // Put an integer
```
Comment (gustavonalle) [inline infinispan-remote/encoding/src/main/java/org/infinispan/tutorial/simple/encoding/InfinispanEncodingCaches.java:25]: It is a bit strange to create a text cache and store integers. Maybe you could use phrases instead, with different charsets (chinese, ascii, etc)?

Comment (karesti) [reply infinispan-remote/encoding/src/main/java/org/infinispan/tutorial/simple/encoding/InfinispanEncodingCaches.java:25]: the idea is to show that even if I try an integer in java it is stored as string. but I'm more than open to suggestions for this example

Comment (gustavonalle) [reply infinispan-remote/encoding/src/main/java/org/infinispan/tutorial/simple/encoding/InfinispanEncodingCaches.java:25]: ok, no prob. In any case, ```text/plain``` is not stored as a String, it should be stored as a ```byte[]```, associated with a ```charset```, e.g. ```text/plain; charset=us-ascii``` (default is ```UTF-8```)

**Comment by pruivo** [comment]
there is a compile error in CI

**Comment by karesti** [comment]
@pruivo would you mind merging plz ?

**Pull request merged**